src/js/tabs/trust.js

```diff
@@ -27,7 +27,11 @@ TrustTab.prototype.angular = function (module)
                                   function ($scope, books, $timeout, $routeParams, id,
                                             $filter, $network, $rpTracker, keychain)
   {
-    $scope.advanced_feature_switch = Options.advanced_feature_switch;
+    
+    updateAdvancedModeSwitch();
```

Comment:
Wrap this with ```$scope.$on('!$scope.loadingAccount && !$scope.account.Balance && connected', function () { ...}``` to wait until the blob is available.

See https://github.com/rht/ripple-client/commit/33b27331b08cc965f5c12e4eaf45485732a77298